templates/conf-mesos-slave.j2

```diff
@@ -5,3 +5,8 @@ export MESOS_CONTAINERIZERS="{{ mesos_containerizers }}"
 export MESOS_EXECUTOR_REGISTRATION_TIMEOUT="{{ mesos_executor_timeout }}"
 export MESOS_PORT="{{ mesos_slave_port }}"
 export MESOS_WORK_DIR="{{ mesos_work_dir }}"
+
+# Additional configs
+{% for config_item in mesos_slave_additional_configs %}
+export MESOS_{{config_item.name}}={{config_item.value}}
```

Comment:
yeah short & flexible (we use such loop over list/dict in a number of roles).
ps: in this case the exact same feature existed already for the general mesos config (and I noticed only during a rebase conflict in the default vars): https://github.com/AnsibleShipyard/ansible-mesos/blob/master/templates/conf-mesos.j2